packages/web-components/src/search-widget/client/components/pagination.svelte

```diff
@@ -0,0 +1,92 @@
+<script>
+  import { onDestroy } from 'svelte'
+  import { getProperties } from '../api/properties'
+  import { getPropertyImages } from '../api/property-images'
+  import searchWidgetStore from '../core/store'
+
+  let totalPage = 1
+  let pageNumber = 1
+
+  const unsubscribeSearchWidgetStore = searchWidgetStore.subscribe(store => {
+    totalPage = store.totalPage
+    pageNumber = store.pageNumber
+  })
+
+  onDestroy(() => {
+    unsubscribeSearchWidgetStore()
+  })
+
+  const handlePageChangeClick = (pageNumber) => async() => {
+    searchWidgetStore.update(values => ({
+      ...values,
+      isLoading: true,
+      resultsMessage: '',
+      properties: [],
+      propertyImages: {},
+      selectedMarker: null,
+      selectedProperty: null,
+      searchType: values.isRental ? 'Rent' : 'Sale',
+    }))
+    const properties = await getProperties($searchWidgetStore.searchKeyword, $searchWidgetStore.isRental, $searchWidgetStore.initializers.apiKey, pageNumber)
+    const propertyImages = await getPropertyImages(properties._embedded, $searchWidgetStore.initializers.apiKey)
+    if (properties && properties._embedded.length) {
+      const propertiesArray = properties._embedded
+      searchWidgetStore.update(values => ({
+        ...values,
+        isLoading: false,
+        properties: propertiesArray,
+        pageNumber,
+      }))
+    }
+
+    if (propertyImages) {
+      searchWidgetStore.update(values => ({
+        ...values,
+        propertyImages,
+      }))
+    }
+  }
+
+
+
+</script>
+
+<style>
+  .pagination {
+  display: inline-block;
+}
+
+.pagination a {
+  color: black;
+  float: left;
+  padding: 8px 16px;
```

Comment:
Cam we use ems please so will re-scale with the font size when themed eg 0.5em 1em